### ISSUE TITLE
Souligner les liens hypertexte dans le corps des articles

### ISSUE THREAD

**Issue by Fuzariane** [c3rb-org/template_RGAA_C3rb#23]
Revoir la couleur des liens dans les corps de texte car ne passent pas le test du ratio de contrast. "Text failed at Level AAA" ou mettre un souligné par défaut, comme cela il est signalé autrement que par la couleur.

RGAA3:
Critère 10.6 [A] Dans chaque page Web, chaque lien dont la nature n'est pas évidente est-il visible par rapport au texte environnant ?
Test 10.6.1 : Dans chaque page Web, chaque lien texte signalé uniquement par la couleur et dont la nature n'est pas évidente a-t-il un rapport de contraste supérieur ou égal à 3:1 par rapport au texte environnant ?

**Comment by cr2a-graphique** [comment]
oula, grosse question !

**Comment by cr2a-graphique** [comment]
re patch par : badcd6bb61cbe44785fb843cd499ff05f46bce10
